### PR TITLE
Normalize visual format for mixin and function options

### PR DESCRIPTION
**Pull request by whmii**
New-line for each input variable

### CODE REVIEW

File: app/assets/stylesheets/functions/_grid-calc.scss [hunk 1]
```diff
@@ -1,6 +1,9 @@
 // Calculates the width of a column for grid-column() and other mixins
 
-@function grid-calc($columns, $collapse:false) {
+@function grid-calc(
+    $columns,
+    $collapse:false
```
Comment (tysongach) [inline app/assets/stylesheets/functions/_grid-calc.scss]: Add a space before `false`

File: app/assets/stylesheets/functions/_grid-column-ratio.scss [hunk 2]
```diff
@@ -1,6 +1,8 @@
 // Calculates the ratio of a given column over its container.
 
-@function grid-column-ratio($columns) {
+@function grid-column-ratio(
+    $columns
+  ) {
```
Comment (tysongach) [inline app/assets/stylesheets/functions/_grid-column-ratio.scss]: I think with only _one_ argument, it makes sense to be on one line…thoughts?

Comment (wardpenney) [inline app/assets/stylesheets/functions/_grid-column-ratio.scss]: Yeah it is easier to read by default, but long ones, the multi-line is better

**Comment by whmii** [comment]
Updated based on coments

**Pull request merged**